main.py

```diff
@@ -38,6 +38,9 @@
 import telescope.utils
 
 
+MAX_THREADS_NORMAL_MODE = 18
+MAX_THREADS_BATCH_MODE = 100
```

Comment:
This is kind of weird. This shows up in both 5bf17f9 (expected) and bf48a14 (unexpected) even though the second one isn't a merge. Any idea what happened here?